src/vivarium_public_health/disease/model.py

```diff
@@ -38,6 +38,21 @@ def __init__(self, cause, initial_state=None, get_data_functions=None, cause_typ
     def name(self):
         return f"disease_model.{self.cause}"
 
+    @property
+    def state_names(self):
+        return [s.name.split('.')[1] for s in self.states]
+
+    @property
+    def transition_names(self):
+        states = {s.name.split('.')[1]: s for s in self.states}
```

Comment:
Why does this need to be a dict containing both the state name and the state itself? It seem like you are only using the value of the dict, so isn't it sufficient to iterate over `self.states`?